app/src/androidTest/java/com/reach5/identity/sdk/demo/MainActivityTest.kt

```diff
@@ -296,6 +296,30 @@ class MainActivityTest {
         )
     }
 
+    @Test
+    fun testSuccessfulGetProfile() = clientTest { client ->
+        val theProfile = aProfile()
```

Comment:
`profile` to stay homogenous 
(there's another test also using `theProfile`, could be renamed)

Reply:
I've named it `theProfile` because it's in conflicts with the `profile` scope.

Reply:
c'est pas mieux renommer `profile` en `profileScope` ?

Reply:
See with @axuan25 😛